va/va_dec_avs.h

```diff
@@ -0,0 +1,144 @@
+ * of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+ * OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT.
+ * IN NO EVENT SHALL INTEL AND/OR ITS SUPPLIERS BE LIABLE FOR
+ * ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+/**
+ * \file va_dec_avs.h
+ * \brief The AVS decoding API
+ *
+ * This file contains the \ref api_dec_avs "AVS decoding API".
+ */
+
+#ifndef VA_DEC_AVS_H
+#define VA_DEC_AVS_H
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * \defgroup api_dec_avs AVS decoding API
+ *
+ * This AVS decoding API supports Jizhun & Guangdian profiles.
+ *
+ * @{
+ */
+
+typedef enum {
+    VA_AVS_I_IMG  = 0,
+    VA_AVS_P_IMG  = 1,
+    VA_AVS_B_IMG  = 2,
+} VAAVSPicType;
+
+typedef struct _VAPictureAVS {
+    VASurfaceID     surface_id;
+    uint32_t        flags;
+    uint32_t        poc;
+
+    /** \brief Reserved bytes for future use, must be zero */
+    uint32_t        va_reserved[VA_PADDING_LOW];
+} VAPictureAVS;
+
+/* flags in VAPictureAVS could be OR of the following */
+#define VA_PICTURE_AVS_INVALID          0x00000001
+#define VA_PICTURE_AVS_TOP_FIELD        0x00000002
+#define VA_PICTURE_AVS_BOTTOM_FIELD     0x00000004
+
+/**
+ * \brief AVS Decoding Picture Parameter Buffer Structure
+ *
+ * This structure conveys picture level parameters and should be sent once
+ * per frame.
+ */
+typedef struct _VAPictureParameterBufferAVS {
+    VAPictureAVS    curr_pic;
+    VAPictureAVS    ref_list[2];
+
+    uint16_t width;                                             // coded width
+    uint16_t height;                                            // coded height
+
+    uint16_t picture_type                           :  2;       // VAAVSPicType
+    uint16_t progressive_seq_flag                   :  1;
+    uint16_t progressive_frame_flag                 :  1;
+    uint16_t picture_structure_flag                 :  1;       // 0:field coding, 1:frame coding
+    uint16_t fixed_pic_qp_flag                      :  1;
+    uint16_t picture_qp                             :  6;
+    uint16_t loop_filter_disable_flag               :  1;
+    uint16_t skip_mode_flag_flag                    :  1;
+    uint16_t picture_reference_flag                 :  1;
+    uint16_t reserved_flag_1                        :  1;       // no_forward_reference_flag
+
+    int8_t   alpha_c_offset;
+    int8_t   beta_offset;
+
+    struct {
+        uint16_t guangdian_flag                     :  1;
+        uint16_t aec_flag                           :  1;
+        uint16_t weight_quant_flag                  :  1;
+        uint16_t reserved_flag_2                    :  1;       // pb_field_enhanced_flag
```

Comment:
same as above

Reply:
Some flag seems never used according to referenced decoder. Any how, seems spec has defined them, they can be defined explicit.